.github/workflows/ci_workflows.yml

```
@@ -110,3 +108,34 @@ jobs:
         - name: (Allowed Failure) Python 3.12 with remote data and dev version of key dependencies
           linux: py312-test-devdeps
           posargs: --remote-data=any --verbose
+
+  report_diff_test_coverage:
+    needs: [ tests ]
+    if: always() && github.event_name == 'pull_request'
+    name: report diff test coverage
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3
```

Should we be more specific?
```suggestion
          python-version: '3.12'
```